app/assets/stylesheets/application.css

```diff
@@ -71,6 +72,8 @@ th h3, th p {
     background-color: #fff;
 }
 
+about.countainer
```

Comment:
Yes I would love help with that :)

Sent from my iPhone

> On Jul 27, 2015, at 2:43 PM, Jason Shultz <notifications@github.com> wrote:
> 
> In app/assets/stylesheets/application.css:
> 
> > @@ -71,6 +72,8 @@ th h3, th p {
> >      background-color: #fff;
> >  }
> >  
> > +about.countainer
> We could do it without creating a whole new layout. You have the code for the about page in the layout instead of using a yield to go back to the /welcome/about.html.erb.
> 
> There's an easier way to do that then creating a whole new template just for one page. If you like, I can help you with it?
> 
> —
> Reply to this email directly or view it on GitHub.
>